routes/web.php

```diff
@@ -310,6 +310,30 @@
 
     });
 
+    /*
+     * Routes related to leaderboards.
+     */
+    Route::group(['prefix' => 'leaderboards', 'as' => 'leaderboards::'], function () {
```

Comment:
Better to just add the auth middleware to the route group.

Reply:
For now, this is can be the case as there is no public page to view the leaderboards yet. However, when the public page goes live this will need to be changed again.